src/Query/PgsqlQueryBuilder.php

```diff
@@ -37,9 +36,10 @@
 use function preg_replace;
 use function reset;
 use function strpos;
+use function strrpos;
 use function version_compare;
 
-class QueryBuilder extends AbstractQueryBuilder
+final class PgsqlQueryBuilder extends QueryBuilder
```

Comment:
Isn't namespace "Yiisoft\\Db\\**Pgsql**\\Query" enough to disambiguate it?

Reply:
Fixed.